Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py

```diff
@@ -472,20 +492,32 @@ def get_file_reputation(file, threshold=None, status="active,inactive"):
         return
 
     threshold = threshold or DEFAULT_THRESHOLD
-    dbot_context = get_dbot_context(indicator, threshold)
+    file_dbot_mapping = {
+        'value': 'Indicator',
+        'subtype': 'Type',
+        'source': 'Vendor',
+    }
+    dbot_context = get_dbot_context(indicator, threshold, file_dbot_mapping)
+    file_type = get_file_type(indicator)
     file_context = get_file_context(indicator, threshold)
-    threat_file_context = get_threat_generic_context(indicator)
-    threat_file_context['MD5'] = threat_file_context.pop('Address')
+
+    indicator_mapping = INDICATOR_MAPPING.copy()
```

Comment:
Not sure I understand the whole `indicator_mapping` vs `INDICATOR_MAPPING` thing.
Could you explain?
Also, let's make this more readable - having both `indicator_mapping` and `INDICATOR_MAPPING` in our code isn't too readable